plannerbenchmark/tests/test_integration.py

```diff
@@ -0,0 +1,122 @@
+import os
+from subprocess import Popen, DEVNULL
+import yaml
+import numpy as np
+import pytest
+
+
+def verify_resultDirExists():
+    resDirName = "tempResults"
+    resultFolderFound = False
+    for fname in os.listdir(resDirName):
```

Comment:
Clarification: Why is a for-loop needed here? The result directories are being removed after each integration test run in line 115 of this file, so at a time there will only be one result directory for a particular case for a specific timestamp; the older results directories will be deleted at the end of the previous test run. Please correct me if I missed anything here. The same comment applies to other functions in this file where a for-loop is used to loop through all the directories.

This comment also holds for the first version of the integration test where the same logic was used, however, it didn't get noticed then.

Reply:
This is required as the `.gitignore` file is present in this folder.
Maybe, I could write separate function as suggested in : https://stackoverflow.com/questions/7099290/how-to-ignore-hidden-files-using-os-listdir

Reply:
Thanks for pointing that out; I missed it.